rome/src/test/resources/atom_1.0.xml

```diff
@@ -17,6 +17,7 @@
     <uri>http://example.com</uri>
     <email>author1@example.com</email>
   </contributor>
+  <icon>http://example.com/icon</icon>
```

Comment:
Looks like we don't assert this value anywhere in the tests. So, the change is not tested?